onnxmltools/convert/sklearn/operator_converters/SVD.py

```diff
@@ -9,7 +9,7 @@
 from ...common._registration import register_converter
 
 
-def convert_truncated_svd(scope, operator, container):
+def convert_truncated_svd(scope, operator, container, flag=False):
```

Comment:
Why creating a special case for FeatureUnion?

Reply:
FeatureUnion would in turn invoke these converters and they need to return back the transformed tensor names (hence flag) back to the FeatureUnion converter where they can be concatenated and the output could be populated.

Reply:
I would prefer not to change the function signature but to create a new operator which overwrites inputs and outputs and then call the converter without any change. The flag parameter does not scale over all converters unless we add it anywhere. The duplication of code is ok for such simple converter but it might be difficult to maintain for bigger converters.

Reply:
Agree with @xadupre.
Even there will be a large code dup, you can refactor it as a common function.

Reply:
Fixed.